packages/tailwindcss-language-server/src/util/v4/design-system.ts

```diff
@@ -58,6 +59,28 @@ function createLoader<T>({
 
       return await jiti.import(url.href, { default: true })
     } catch (err) {
+      // If the request was to load a first-party plugin and we can't resolve it
+      // locally, then fall back to the built-in plugins that we know about.
+      if (resourceType === 'plugin' && id in plugins) {
+        console.log('Loading bundled plugin for: ', id)
```

Comment:
Did you keep these logs on purpose?

Reply:
Yep. We send these to the "Tailwind CSS IntelliSense" output panel in VSCode. It's useful for triaging issues.